lib/neo4j/railtie.rb

```diff
@@ -74,8 +75,10 @@ def final_config!(neo4j_config)
     end
 
     def support_deprecated_session_configs!(neo4j_config)
-      ActiveSupport::Deprecation.warn('neo4j.config.sessions is deprecated, please use neo4j.config.session (not an array)') if neo4j_config.sessions.present?
-      neo4j_config.session ||= (neo4j_config.sessions && neo4j_config.sessions[0]) || {}
+      if neo4j_config.sessions.present?
```

Comment:
;(  Huh, when I refactored things (I don't recall when) I thought I made it so that `neo4j_config` doesn't have a `sessions` property anymore but rather the `session` property is set to `sessions[0]`.  That refactor might have happened after this PR.  I believe we made some more changes recently so you might want to merge `master` in